src/opnsense/scripts/unbound/blocklists.py

```diff
@@ -141,12 +141,36 @@ def uri_reader(uri):
                     'blocklist download %(uri)s (lines: %(lines)d exclude: %(skip)d block: %(blocklist)d)' % file_stats
                 )
 
+        # fetch configured DNS response type
+        localdata = None
```

Comment:
None cannot really happen… tri-state Boolean?

Reply:
Taking advantage of the fact that False and 0 do the same things.  I've made this explicitly 0 in the next update.